### ISSUE TITLE
Unable to find TSDB installer for SLES-15

### ISSUE THREAD

**Issue by poojabms** [timescale/timescaledb#1633]
Hello everyone,

We are considering TSDB for supporting events in our production environment and need some help with the same. On the official site of TSDB I see installers available for ubuntu and redhat, in addition to other platforms. However I don't see one for SLES-15. 
https://docs.timescale.com/latest/getting-started/installation
Could some one help?
NOTE: Installation using source code or on docker is not an option for us.

Any help will be appreciated.

Thanks,
Pooja

**Comment by mfreed** [comment]
Hi Pooja, we currently don't have any immediate plans to increase the number of different distributions that we build for TimescaleDB.  But I'll leave this Issue open to try to gauge some of the demand.

**Comment by poojabms** [comment]
Thanks much Mike.  Afraid I'll need to look out for options now :(

**Comment by tigerfoot** [comment]
@poojabms did you try the package available on obs ?
[timescaledb](https://build.opensuse.org/package/show/server:database:postgresql/timescaledb) 
Actually it lack some version behind (1.4.2 available for 9.6,10,11) cause I forget (and was waiting pg 12 support) to update it.
It is build to respect SUSE way of life with postgresql. Of course it can only support Apache2 modules on the public build system. But it can also be a good start if you need to build more modules. Timescale can also get in touch with SUSE to have the other part build in the non-free system.

**Comment by Kazmirchuk** [comment]
+1, would be good to have TSDB on OpenSUSE LEAP

**Comment by fwegmann** [comment]
I'd definitely chime in here. SUSE is an important distribution over here in Europe, and, e.g., the preferred one over here at our company. I'm also currently planning a solution around TimescaleDB and it would be of tremendous value to have a turn-key installation of the current version right at hand.
I guess, it's not especially rocket science to update TSDB to fit into SUSE's package landscape. The current distro SLES 15SP3 already provides PostgreSQL14, but only for version 12 there is also a timescaledb package.

**Comment by tigerfoot** [comment]
Contributing is often a key attitude in open source world. Maybe you can help like proposing to co-maintain the package @build.opensuse.org, so I would not be alone to maintain this one on my "too short spare time". Try to convince or motivated me, or take the lead the package is available for all :-)

**Comment by fwegmann** [comment]
Fair enough! If I weren't deep in other open source projects besides regular work, I would/could consider it. I'm sorry, if I couldn't assess the current situation in the timescale project correctly. in that you're apparently lacking manpower for this kind of tasks. I understand perfectly well the situation of many open projects that are driven by far too few devoted people.

**Comment by Tanushreesuresh** [comment]
how to install in suse terminal

**Comment by tigerfoot** [comment]
> how to install in suse terminal

well you start at the start by lurcking what is available https://software.opensuse.org/search?baseproject=ALL&q=timescaledb

then choose your variant.

**Comment by alfista-sk** [comment]
Hi,
I try this to but on SUSE Server enterprise 15SP7. I have found that you have implemented the TSDB to the original SUSE packages, but there is only Apache 2.0 license and I need to use community (timescale) edition, while I need to use the compression.
Is there any possibility how I can get this?
Thanks.

**Comment by Kazmirchuk** [comment]
I'm not affiliated with the TSDB team, but (to correct my comment above from 2021) once I switched to Docker/podman, I never looked back to installing TSDB from packages

**Comment by alfista-sk** [comment]
OK understand, thanks. I don't work now in docker.
